mentat/terminal/client.py

```diff
@@ -194,7 +194,7 @@ def run(self):
         asyncio.run(self._run())
 
 
-def run_cli():
+def get_parser():
```

Comment:
Why is this in this PR? Let's not change anything outside of CONTRIBUTING.MD in this PR.

Reply:
This is is necessary to get the automated documentation working for the command line - generating the parser needs to be in a separate function from using the parser.

Reply:
It's so the parser fields can be generated in the documentation. I actually like the non-CONTRIB changes in this PR a lot. It seems like a good step towards having a place for user documentation. Though I can see the logic of splitting the pr up

Reply:
Ah, makes sense. Thanks for explaning!